src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -139,6 +139,10 @@ public function getConfigTreeBuilder()
                                 }
 
                                 if (false !== strpos($v, '/')) {
+                                    if ('0.0.0.0/0' === $v) {
+                                        return false;
+                                    }
```

Comment:
I would move this above the earlier `if` (no need to do the `strpos()` check first.